frontend/src/components/camp/Basic.vue

```diff
@@ -3,24 +3,15 @@ Displays details on a single camp and allows to edit them.
 -->
 
 <template>
-  <v-card>
-    <v-toolbar dense color="blue-grey lighten-5">
-      <v-icon left>
-        mdi-cogs
-      </v-icon>
-      <v-toolbar-title>
-        Einstellungen
-      </v-toolbar-title>
-      <v-spacer />
-    </v-toolbar>
+  <card-view title="Admin">
```

Comment:
Mal als Vorwarnung: Das wird den ein oder anderen Merge-conflict geben, je nach dem wer schneller ist :-)
Beim Pull request von Forte und mir gibts Basic.vue nicht mehr:
https://github.com/ecamp/ecamp3/pull/382/files

Reply:
Von mir aus kann ich auch nach euren Pull Requests mergen. Dann bringe ich das in die gewünschte Form.